discovery-provider/src/tasks/generate_trending.py

```diff
@@ -141,6 +154,13 @@ def generate_trending(db, time, genre, limit, offset):
             else:
                 track_entry[response_name_constants.repost_count] = 0
 
+            # Populate repost counts with respect to time
+            time_key = get_repost_time_key(time) + "_repost_count"
```

Comment:
while it's nice to have the time range as part of the key, it ends up being more work on the client because we have to change what fields we read from.

I would prefer we just do something like windowed_repost_count / repost_count_trailing / repost_count_timerange or something like that